portfolio/src/main/webapp/interests.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Interests</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="content">
+      <div class="header">
+      <h1>Anthony Zhu</h1>
+
+      <a id="img-link" href="https://www.linkedin.com/in/anthony-zhu-738215175/" target="_blank">
+        <img id="img-link" src="/images/LI-In-Bug.png" alt="LinkedIn Profile" style="width:32px;height:32px;border:0;">
+      </a>
+
+      <a id="img-link" href="https://github.com/anthonyqzhu" target="_blank">
+        <img id="img-link" src="/images/GitHub-Mark-32px.png" alt="GitHub Profile" style="width:32px;height:32px;border:0;">
```

Comment:
Here and elsewhere, don't use style= and instead stick all the styling should go in the CSS stylesheet. We want to keep the HTML to just the semantics, while the CSS stylesheet controls all the presentation.